byzcoin/bcadmin/main.go

```diff
@@ -65,13 +64,28 @@ var cmds = cli.Commands{
 				EnvVar: "BC",
 				Usage:  "the ByzCoin config to use",
 			},
-			cli.StringFlag{
+			cli.StringSliceFlag{
 				Name:  "identity",
-				Usage: "the identity of the signer who will be allowed to access the contract (e.g. ed25519:a35020c70b8d735...0357))",
+				Usage: "the identity(ies) of the signer(s) who will be allowed to access the contract (e.g. ed25519:a35020c70b8d735...0357))",
+			},
+			cli.StringFlag{
```

Comment:
I wrote to one of the students also working on this to simply use a `string` here, as the user can then do whatever he wants with regard to the identity.

Reply:
I don't understand this comment...